README.md

```diff
@@ -7,12 +7,27 @@ cargo-geiger ☢️
 [![crates.io](https://img.shields.io/crates/v/cargo-geiger.svg)](https://crates.io/crates/cargo-geiger)
 [![Crates.io](https://img.shields.io/crates/d/cargo-geiger?label=cargo%20installs)](https://crates.io/crates/cargo-geiger)
 
-A program that lists statistics related to the usage of unsafe Rust code in a Rust
+A tool that lists statistics related to the usage of unsafe Rust code in a Rust
 crate and all its dependencies.
 
+These statistics currently provide blunt information to aid auditing, e.g:
+* https://github.com/rust-secure-code/safety-dance
+* https://github.com/crev-dev/cargo-crev
+
+This tool is currently not meant to advise directly whether the code ultimately is truly unsound or not.
+
+This project is intended as an attempt to:
+* Create pressure against __unnecessary__ usage of unsafe Rust
+
+This project is not intended as an attempt to:
+* Create pressure against __necessary__ usage of unsafe Rust
+
+There are several proposals / projects to qualify the statistics in more end-user usable form:
+* Safety Cross-Checking Human Validations - https://github.com/rust-secure-code/cargo-geiger/discussions/303
+
 This cargo plugin were originally based on the code from two other projects:
```

Comment:
```suggestion
This Cargo plugin was originally based on the code from two other projects:
```

May as well fix the previous README's grammar here

Reply:
Thanks!